beckon-mesh/src/main/java/com/technocreatives/beckon/mesh/scenario/TestCases.kt

```diff
@@ -56,7 +56,7 @@ sealed interface Test {
 
     data class Action(val execute: suspend () -> Either<TestFailed, Unit>) : Test
 
-    data class Repeat(val times: Int, val test: Test) : Test
```

Comment:
why `SuccessRate` here? The `Repeat` is supposed to repeat any kind of tests?

Reply:
We are doing SuccessRate test but we are only testing on 2 lights so I just wanted to repeat that couple of times so we can see the successRate after multiple calls

Reply:
Oh yes, I need to change it back to Test. My mistake, fixing it 👍